src/mantarray_desktop_app/workers/magnet_finder.py

```diff
@@ -18,7 +17,7 @@ def run_magnet_finding_alg(
     result_dict: Dict[str, Any],
     recordings: List[str],
     output_dir: Optional[str] = None,
-    end_time: Union[float, int] = np.inf,
+    end_time: Union[float, int] = None,
```

Comment:
I think you're going to need to make this `Optional[Union[float, int]]` or mypy will complain